src/QuickMarc.js

```diff
@@ -24,40 +20,41 @@ const QuickMarc = ({
   onClose,
   onSave,
 }) => {
+  const location = useLocation();
+
+  const permissionsMap = {
+    'create-bibliographic': 'ui-quick-marc.quick-marc-editor.create',
+    'edit-bibliographic': 'ui-quick-marc.quick-marc-editor.all',
+    'derive-bibliographic': 'ui-quick-marc.quick-marc-editor.derive.execute',
+    'create-authority': 'ui-quick-marc.quick-marc-authorities-editor.create',
+    'edit-authority': '', // ui-quick-marc.quick-marc-authorities-editor.all
```

Comment:
is this comment intentional?

Reply:
This permission was commented in the previous implementation, and I decided to leave it too.